Sophos/sophos-xdr/ingest/parser.yml

```
@@ -0,0 +1,53 @@
+name: sophos-xdr
+pipeline:
+  - external:
+      name: json.parse-json
+      properties:
+        input_field: "{{original.message}}"
+        output_field: message
+    name: parse_json
+
+  - name: set_ecs_fields
+  - name: set_sophos_fields
+
+stages:
+  set_ecs_fields:
+    actions:
+      - set:
+          "@timestamp": "{{parse_json.message.calendar_time}}"
+
+          event.kind: "event"
+          event.ingested: "{{parse_json.message.ingestion_timestamp}}"
+          event.severity: "{{parse_json.message.ioc_severity}}"
+
+          user.name: "{{parse_json.message.meta_username}}"
+
+          file.path: "{{parse_json.message.path}}"
```

According to the test, It seems to be the [registry path](https://www.elastic.co/guide/en/ecs/8.6/ecs-registry.html#field-registry-path).
I suggest adding the following registry fields:
```suggestion
          registry.path: "{{parse_json.message.path}}"
          registry.key: "{{parse_json.message.key}}"
          registry.data.type: "{{parse_json.message.type}}"
          registry.data.strings: "['{{parse_json.message.data}}']"
```